src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ExpressionHelper.cs

```diff
@@ -19,13 +19,16 @@ public static string GetExpressionText(string expression)
             return string.Equals(expression, "model", StringComparison.OrdinalIgnoreCase) ? string.Empty : expression;
         }
 
-        public static string GetExpressionText(LambdaExpression expression)
+        public static bool GetExpressionText(LambdaExpression expression, out string expressionText)
         {
             if (expression == null)
             {
                 throw new ArgumentNullException(nameof(expression));
             }
 
+            expressionText = string.Empty;
+            var isMemberAccessExpressionOnly = true;
```

Comment:
Suggest `var containsIndexers = false;` since this will remain `true` for variable access as well.